test/tests/test_harness/extra_info

```diff
@@ -0,0 +1,27 @@
+[Tests]
+  [./extra_info_caveats]
+    type = RunApp
+    input = good.i
+    compiler = 'GCC'
+    method = 'OPT OPROF'
+    petsc_version = '3.7.5'
+    petsc_version_release = true
+    petsc_debug = 'TODO'
+    slepc_version = '3.7.4'
+    library_mode = 'TODO'
+    mesh_mode = 'distributed'
+    dtk = 'TODO'
+    unique_ids = 'TODO'
+    vtk = 'TODO'
+    tecplot = 'TODO'
+    dof_id_bytes = 'TODO'
+    curl = 'TODO'
+    tbb = 'TODO'
+    superlu = 'TODO'
+    slepc = 'TODO'
+    unique_id = 'TODO'
+    cxx11 = 'TODO'
+    asio = 'TODO'
+    boost = 'TODO'
+  [../]
+[]
```

Comment:
Even though they say 'TODO' they do work... I forgot to change the name. Well, actually I didn't know what to put in all those fields... true? A version number? Anyways, it doesn't really matter. The caveat _is_ being displayed.

Reply:
Hmm...  Looks like we need to start cleaning these up! We don't need cxx11 anymore for instance. Not sure about asio.

Reply:
I really think we should fill these in with values... I know you don't want to stretch this out, but if I merge this, it'll probably be in there for awhile. What I'd do is search for "tests" files and grep for each one of these. It seems like it won't take long to fill them in. After that, we'll clean them up (another ticket).